README.md

```diff
@@ -243,18 +243,18 @@ $ mkdir -p autosim_nmpc
 $ git clone -b nmpc_panoc https://github.com/vittoriocataffo/A-Nonlinear-Model-Predictive-Control-Strategy-for-Autonomous-Racing-of-Scale-Vehicles.git
 $ cd PANOC_DYNAMIC_MOTOR_MODEL
 $ python3 all_wheel_drive_PANOC_DINAMYC_motor_model.py
-$ python3 PANOC_DINAMYC_motor_model.py
+<!--- $ python3 PANOC_DINAMYC_motor_model.py --> 
 $ cd ../OBS_PANOC_DYNAMIC_MOTOR_MODEL
 $ python3 OBS_all_wheel_drive_PANOC_DINAMYC_motor_model.py
-$ python3 OBS_PANOC_DINAMYC_motor_model.py
+<!--- $ python3 OBS_PANOC_DINAMYC_motor_model.py -->
```

Comment:
This is to build the NMPC with the Tatulea's parameters. I think leaving it here might be misleading